README.md

```diff
@@ -114,13 +161,13 @@ immediately; there is no need to call `logger.flush()`.
 | ------------------- | ---------- | -------------- | ---------------------------------------------------------------------------- |
 | `global`            | `object`   |                | Options for global console logger (outside forked jobs)                      |
 | `global.threshold`  | `number`   | `Logger.INFO`  | The minimum level a log entry must have to be written to the console         |
-| `fork`              | `object`   |                | Options for forked job logger (the one that generates HTML dumps)            |
-| `fork.logDir`       | `string`   |                | Root directory for generated logs; this is the only required option          |
-| `fork.threshold`    | `number`   | `Logger.ERROR` | The minimum level a log entry must have to mark a job queue for writing      |
-| `fork.cloneData`    | `boolean`  | `false`        | Clone data of log entries using V8 serialize / deserialize                   |
-| `fork.gc`           | `object`   |                | Garbage collection options:                                                  |
-| `fork.gc.interval`  | `number`   | `60`           | How often GC will be run, in seconds                                         |
-| `fork.gc.threshold` | `number`   | `300`          | How long since a logger has last been touched before GC should auto-flush it |
+| `runTask`              | `object`   |                | Options for forked job logger (the one that generates HTML dumps)            |
+| `runTask.logDir`       | `string`   |                | Root directory for generated logs; this is the only required option          |
+| `runTask.threshold`    | `number`   | `Logger.ERROR` | The minimum level a log entry must have to mark a job queue for writing      |
+| `runTask.cloneData`    | `boolean`  | `false`        | Clone data of log entries using V8 serialize / deserialize                   |
+| `runTask.gc`           | `object`   |                | Garbage collection options:                                                  |
+| `runTask.gc.interval`  | `number`   | `60`           | How often GC will be run, in seconds                                         |
+| `runTask.gc.threshold` | `number`   | `300`          | How long since a logger has last been touched before GC should auto-flush it |
```

Comment:
potřebujem vůbec nějaký společný options pro forknutý task-specific handlery? nebylo byl lepší tohle předat prostě rovnou tomu html handleru?

Reply:
To je pozustatek v Readme... V kodu se zadne takove options nepredavaji